### ISSUE TITLE
🐞 Bug report: Webhook error when response code is 201

### ISSUE THREAD

**Issue by pointmax** [elementor/elementor#19502]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi, I am using a Webhook in the Elementor Pro contact form. The webhook responds with the status code 201 (created). This Throws an error because Elementor only checks for status code 200 and throws an exception otherwise.

![image](https://user-images.githubusercontent.com/49361135/184895404-8813ae0d-1c49-4b1c-9a6a-21a8dbc161f5.png)

### Steps to reproduce

Create a webhook that responds with status code 201. Set it as a webhook for a contact form. Submit the form.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.21.6
	MySQL version: mariadb.org binary distribution v10.6.5
	PHP Version: 8.0.14
	PHP Memory Limit: 500M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0.1
	Site URL: https://staging-web.notarity.com
	Home URL: https://staging-web.notarity.com
	WP Multisite: No
	Max Upload Size: 100 MB
	Memory limit: 40M
	Max Memory limit: 500M
	Permalink Structure: /blog/%postname%/
	Language: en-US
	Timezone: Europe/Vienna
	Admin Email: max.pointinger@staging-web.notarity.com
	Debug Mode: Active

== Theme ==
	Name: Notarity Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: de_AT
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.5112.81 Safari/537.36

== Active Plugins ==
	3r Elementor Timeline Widget
		Version: 1.8
		Author: B.M. Rafiul Alam

	All-in-One WP Migration
		Version: 7.63
		Author: ServMask

	All-in-One WP Migration Unlimited Extension
		Version: 2.46
		Author: ServMask

	Better Search Replace
		Version: 1.4.1
		Author: Delicious Brains

	Custom Post Type UI
		Version: 1.12.1
		Author: WebDevStudios

	Disable REST API
		Version: 1.7
		Author: Dave McHale

	Easy WP SMTP
		Version: 1.4.7
		Author: wpecommerce, alexanderfoxc

	Elementor
		Version: 3.6.8
		Author: Elementor.com

	Elementor Blocks for Gutenberg
		Version: 1.0.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.7.3
		Author: Elementor.com

	Favicon by RealFaviconGenerator
		Version: 1.3.25
		Author: Philippe Bernard

	iThemes Security
		Version: 8.1.2
		Author: iThemes

	Matomo Analytics - Ethical Stats. Powerful Insights.
		Version: 4.11.0
		Author: Matomo

	Redirection
		Version: 5.3.2
		Author: John Godley

	TranslatePress - Business
		Version: 1.1.9
		Author: Cozmoslabs, Razvan Mocanu, Madalin Ungureanu

	TranslatePress - Multilingual
		Version: 2.3.4
		Author: Cozmoslabs, Razvan Mocanu, Madalin Ungureanu, Cristophor Hurduban

	User Profile Picture
		Version: 2.6.0
		Author: Cozmoslabs

	W3 Total Cache
		Version: 2.2.4
		Author: BoldGrid

	WebP Express
		Version: 0.25.5
		Author: Bjørn Rosell

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast

	Yoast SEO
		Version: 19.5.1
		Author: Team Yoast

== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Inactive
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Active
	Accessibility Improvements: Active
	Import Export Template Kit: Active by default
	Additional Custom Breakpoints: Inactive by default
	Hide native WordPress widgets from search results: Active by default
	: Inactive by default
	Flexbox Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive
	Landing Pages: Inactive
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Page Transitions: Inactive
	Notes: Active by default
	Form Submissions: Active
	Scroll Snap: Active by default

== Log ==
	
Log: showing 20 of 322022-06-03 15:43:10 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-06-03 15:43:10 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-06-03 15:43:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.0',
  'to' => '3.7.1',
)]
2022-06-03 15:43:10 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.0',
  'to' => '3.7.1',
)]
2022-08-16 14:40:00 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.5',
  'to' => '3.7.1',
)]
2022-08-16 14:40:00 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.1',
  'to' => '3.7.3',
)]
2022-08-16 14:40:10 [info] elementor::elementor_updater Started 
2022-08-16 14:40:10 [info] Elementor/Upgrades - _on_each_version Start  
2022-08-16 14:40:10 [info] Elementor/Upgrades - _on_each_version Finished 
2022-08-16 14:40:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.5',
  'to' => '3.7.1',
)]
2022-08-16 14:40:13 [info] elementor-pro::elementor_pro_updater Started 
2022-08-16 14:40:13 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-08-16 14:40:13 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-08-16 14:40:13 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Start  
2022-08-16 14:40:15 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Finished 
2022-08-16 14:40:15 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-08-16 14:40:15 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-08-16 14:40:15 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Start  
2022-08-16 14:40:17 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Finished 
2022-08-16 14:40:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.1',
  'to' => '3.7.3',
)]

JS: showing 7 of 7JS: 2022-05-06 13:05:32 [error X 5][https://notarity.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.6.4:2:515479] Cannot read properties of null (reading \'getBoundingClientRect\') 
JS: 2022-05-10 08:54:30 [error X 2][https://notarity.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.6.5:2:235643] Cannot read properties of undefined (reading \'id\') 
JS: 2022-05-10 13:00:47 [error X 31][https://notarity.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.6.5:2:450542] T.getContainer is not a function 
JS: 2022-06-17 12:21:58 [error X 25][https://notarity.com/wp-includes/js/jquery/jquery.min.js?ver=3.6.0:2:31703] Failed to execute \'observe\' on \'IntersectionObserver\': parameter 1 is not of type \'Element\'. 
JS: 2022-07-25 14:15:48 [error X 1][https://www.gstatic.com/recaptcha/releases/CHIHFAf1bjFPOjwwi5Xa4cWR/recaptcha__de.js:352:375] reCAPTCHA has already been rendered in this element 
JS: 2022-08-16 12:35:19 [error X 15][https://staging-web.notarity.com/wp-content/plugins/elementor/assets/js/common.min.js?ver=3.6.5:3:62238] elementorDevTools is not defined 
JS: 2022-08-16 12:35:22 [error X 1][https://staging-web.notarity.com/wp-content/plugins/elementor/assets/js/web-cli.min.js?ver=3.6.5:3:153870] Cannot read properties of undefined (reading \'split\') 

PHP: showing 3 of 3PHP: 2022-05-06 17:57:19 [warning X 3018][/var/www/html/wp-content/plugins/elementor-pro/modules/forms/classes/form-record.php::179] Undefined array key "referrer" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-05-09 18:01:17 [warning X 3][/var/www/html/wp-content/plugins/elementor-pro/modules/forms/classes/ajax-handler.php::260] Undefined array key "post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-05-09 18:10:11 [warning X 4][/var/www/html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	3r Elementor Timeline Widget: Compatibility not specified
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

**Comment by jessehanley** [comment]
Running into this too — would be nice to have this fixed and merge in. Have a few customers who we've pushed to Elementor get blocked by this. Thanks!

**Comment by mikeb629** [comment]
Yep same here - getting blocked as well. I've got no control over the response, so no way to fix unless Elementor supports a 201.
